src/network_event/client_event.rs

```diff
@@ -299,15 +313,33 @@ mod tests {
         let mut app = App::new();
         app.add_plugins(MinimalPlugins)
             .add_plugins(ReplicationPlugins.build().disable::<ClientPlugin>())
-            .add_client_event_with::<DummyEvent, _, _>(|| {}, || {})
+            .add_client_event_with::<DummyEvent, _, _>(SendPolicy::Ordered, || {}, || {})
             .update();
     }
 
     #[test]
-    fn sending_receiving() {
+    fn sending_receiving_ordered() {
```

Comment:
I would suggest to remove the added test and keep only one with `SendPolicy::Ordered`. We just passing parameter to Renet, nothing to exclusively test.